src/main/java/seedu/address/logic/exportutil/ExportUtil.java

```diff
@@ -57,31 +56,125 @@ public static void exportDataToFile (String fileIdentifier, String extension, St
      * Formats and export person to file
      */
     public static void exportPerson (Person person) {
-        try {
-            String name = person.getName().toString();
-            String fileIdentifier = name.replaceAll("[^a-zA-Z0-9]", "");;
-            String matricNumber = person.getMatricNumber().toString();
-            String phone = person.getPhone().toString();
-            String email = person.getEmail().toString();
-            String address = person.getAddress().toString();
-            String gender = person.getGender().toString();
-            String yearOfStudy = person.getYearOfStudy().toString();
-            String major = person.getMajor().toString();
-            Set<Tag> tagSet = person.getTags();
-            ArrayList<String> tagList = new ArrayList<String>();
-            if (!tagSet.isEmpty()) {
-                for (Tag t : tagSet) {
-                    tagList.add(t.toString());
-                }
+        String name = person.getName().toString();
+        String fileIdentifier = name.replaceAll("[^a-zA-Z0-9]", "");;
+        String matricNumber = person.getMatricNumber().toString();
+        String phone = person.getPhone().toString();
+        String email = person.getEmail().toString();
+        String address = person.getAddress().toString();
+        String gender = person.getGender().toString();
+        String yearOfStudy = person.getYearOfStudy().toString();
+        String major = person.getMajor().toString();
+
+        /*Set<Tag> tagSet = person.getTags();
```

Comment:
just want to confirm that we aren't exporting the tags assigned to a member?

Reply:
I can uh. But not my focus now hahaha. maybe later